test/unit_tests/aievec_tests/bf16_cast_float/bf16_cast_float.mlir

```diff
@@ -0,0 +1,13 @@
+// REQUIRES: valid_xchess_license
+// RUN: mlir-opt %s --pass-pipeline="builtin.module(func.func(tosa-to-linalg-named, tosa-to-linalg))"| mlir-opt --linalg-fuse-elementwise-ops --eliminate-empty-tensors --empty-tensor-to-alloc-tensor --one-shot-bufferize="allow-return-allocs allow-unknown-ops bufferize-function-boundaries function-boundary-type-conversion=identity-layout-map" --drop-equivalent-buffer-results --buffer-results-to-out-params --buffer-deallocation --canonicalize --cse --convert-linalg-to-affine-loops --affine-super-vectorize="virtual-vector-size=16" | aie-opt --convert-vector-to-aievec="aie-target=aieml" -lower-affine | aie-translate -aieml=true --aievec-to-cpp -o dut.cc
```

Comment:
You might want to move all the pipeline definitions from `test/Integration/lit.local.cfg` to `test/lit.local.cfg`, so that you can do `// RUN: aie-opt %s %tosa-to-linalg% | aie-opt %linalg-to-affine% | FileCheck %s` in your test scripts under `/unit_tests/`. It will be really painful if we ever want to update the script latter in the future and find that the scripts are all over the places.